src/pages/ProductList/ProductList.scss

```
@@ -0,0 +1,201 @@
+@import '../../styles/baseStyle/colors.scss';
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+
+  display: flex;
+  justify-content: center;
+  // box-sizing: inherit;
+}
+
+.productList {
+  box-sizing: border-box;
+  margin-top: 150px;
+  header {
+    .subHeadBottom {
+      h1 {
+        // background-color: lightcoral;
+        margin-top: 30px;
+      }
+      p {
+      }
+    }
+  }
+
+  main {
+    width: 1140px;
+    margin: 0 auto;
+    // justify-content: center;
+    section.headContainer {
+      position: relative;
+      border-bottom: 1px solid lightgrey;
+      article.layoutWrapper {
+        .container {
+          margin-bottom: 35px;
+          padding-top: 40px;
+          button {
+            height: 55px;
+            margin: 0 3px 0 3px;
+            padding: 0 30px 0 30px;
+            border: 1px solid black;
+            font-size: 18px;
+          }
+        }
+        .containerDetail {
+        }
+        .filterButtonWrapper {
+          text-align: right;
+          // background-color: lightgreen;
+          .filterButton {
+            margin-bottom: 20px;
+            span {
+              .filterButtonList {
+              }
+              .filterButtonTxt {
+              }
+              .filterButtonArrow {
+              }
+            }
```

```suggestion
          .filterButton {
            margin-bottom: 20px;
```
아직 스타일이 없는 셀렉터의 경우 삭제해주세요!

---

스타일 사용하지 않는 셀렉터는 삭제하도록하겠습니다 감사합니다🙂